com.unity.render-pipelines.high-definition/Documentation~/Decal-Projector.md

```diff
@@ -14,29 +14,33 @@ The Decal Projector includes a Scene view representation of its bounds and proje
 
 * A box that describes the 3D size of the projector; the projector draws its decal on every Material inside the box.
 
-* An arrow that indicates the direction the projector faces.
+* An arrow that indicates the direction the projector faces. This arrow is on the pivot point.
 
 ![](Images/DecalProjector2.png)
 
-The decal Projector also includes two gizmos that add handles on every face for you to click and drag to alter the size of the projector's bounds.
+The decal Projector also includes three gizmos. The first two add handles on every face for you to click and drag to alter the size of the projector's bounds.
 
 |**Button**|**Gizmo**|**Description**|
 |-----|-----|-----|
-|![](Images/DecalProjector3.png)|**Scale**|Scales the decal with the projector box. This changes the UVs of the Material to match the size of the projector box. This stretches the decal.|
-|![](Images/DecalProjector4.png)|**Crop**|Crops the decal with the projector box. This changes the size of the projector box but not the UVs of the Material. This crops the decal.|
+|![](Images/DecalProjector3.png)|**Scale**|Scales the decal with the projector box. This changes the UVs of the Material to match the size of the projector box. This stretches the decal. The Pivot remains still.|
+|![](Images/DecalProjector4.png)|**Crop**|Crops the decal with the projector box. This changes the size of the projector box but not the UVs of the Material. This crops the decal. The Pivot remains still.|
+|![](Images/DecalProjector5.png)|**Pivot / UV**|Move the pivot without moving the projection box. This alter the transform position.<br />Also, set the UV used on the projected texture.|
```

Comment:
Moves the decal's pivot point without moving the projection box. This changes the transform position.<br />Note this also sets the UV used on the projected texture.